### ISSUE TITLE
Some errors have the wrong line number

### ISSUE THREAD

**Issue by myrjola** [willthames/ansible-lint#45]
The examples/example.yml gets the wrong line number 0 for errors like:

```
[ANSIBLE0006] git used in place of git module
[ANSIBLE0004] Git checkouts must contain explicit version
```

```
➜  ansible-lint git:(master) python --version
Python 2.7.6
➜  ansible-lint git:(master) pip list
ansible (1.8.2)
ansible-lint (2.0.4)
ecdsa (0.11)
Jinja2 (2.7.3)
MarkupSafe (0.23)
paramiko (1.15.2)
pip (6.0.6)
pycrypto (2.6.1)
PyYAML (3.11)
RBTools (0.6.3)
setuptools (11.3.1)
vboxapi (1.0)
➜  ansible-lint git:(master) ansible-lint examples/example.yml
[ANSIBLE0006] git used in place of git module
examples/example.yml:0
Task/Handler: executing git through command

[ANSIBLE0006] git used in place of git module
examples/example.yml:0
Task/Handler: executing git through command

[ANSIBLE0001] Old style (${var}) brackets
examples/example.yml:10
    action: command echo ${oldskool}

[ANSIBLE0004] Git checkouts must contain explicit version
examples/example.yml:0
Task/Handler: git check

[ANSIBLE0004] Git checkouts must contain explicit version
examples/example.yml:0
Task/Handler: git check 2

[ANSIBLE0004] Git checkouts must contain explicit version
examples/example.yml:0
Task/Handler: using git module

[ANSIBLE0003] Mismatched { and }
examples/example.yml:13
    action: debug oops a missing {{bracket}

[ANSIBLE0002] Trailing whitespace
examples/example.yml:19
    action: do nothing

[ANSIBLE0002] Trailing whitespace
examples/example.yml:41
    with_items:
```

**Comment by myrjola** [comment]
I bisected this issue to the following commit:

```
commit 6bd287c7ca561c1f7c8782d7cbead5497c899bd5
Author: willthames <will.thames@suncorp.com.au>
Date:   Tue Jul 8 10:48:24 2014 +1000

    Major refactor of matchblock -> matchtask

    matchtask is more aware of the action being taken
    tasks are normalized to all contain an action with a module
    and associated arguments.

    Added tests for the new `utils.normalize_task` method
    Refactored `CommandsInstead*` rules to use matchtask, fixing #16.

    Moved `TaskHasTag` rule from default rules into examples as it
    is not a core requirement.

    More PEP8 fixes
```

**Comment by willthames** [comment]
Fixed by #46
